modules/zivid/hdr.py

```diff
@@ -1,6 +1,7 @@
 """Contains a global HDR method."""
 import _zivid
 from zivid.frame import Frame
+import zivid._settings_converter as _settings_converter
```

Comment:
I don't think you need to do this manually, did you try to use the zivid.Settings ctor and passsing in _zivid.Settings?

Reply:
Actually, here it is used to go from public ones to internal ones, see the line below:
` _settings_converter.to_internal_settings(settings)`
there seems to no ctor which would allow _zivid.Settings(zivid.Settings).

Reply:
You are correct. Keep it as is. I see the same is done in `zivid.Camera.settings`.